### PR TITLE
Bug Fixes

### LINKED ISSUE #760: Bugs
Invalid argument supplied error related to the `$essay->content` attribute. Only getting this error on influencer applications and not testimonies even though they are both `Essay` objects.

screen shot 2017-09-21 at 11 48 02 am

- [x] RESOLVED

### PR DESCRIPTION
**Pull request by jgiovanni**
#760

### CODE REVIEW

File: resources/assets/js/components/records/medicals/medical-create-update.vue [hunk 1]
```diff
@@ -550,7 +550,7 @@
         },
         mounted(){
             if (this.isUpdate) {
-                this.$http.get('medical/releases/' + this.id, { params: { include: 'conditions,allergies,uploads,user'} }).then((response) => {
+                this.$http.get(`medical/releases/${this.id}`, { params: { include: 'conditions,allergies,uploads,user'} }).then((response) => {
```
Comment (nkeena) [inline resources/assets/js/components/records/medicals/medical-create-update.vue]: Is this valid syntax?

Comment (jgiovanni) [reply resources/assets/js/components/records/medicals/medical-create-update.vue]: yup, https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

Comment (jgiovanni) [reply resources/assets/js/components/records/medicals/medical-create-update.vue]: I noticed that the ID is not being passed in the blade file

**Comment by nkeena** [comment]
@jgiovanni , I see you're still adding to this PR. When it's ready to merge, please label it as Review Needed. Thanks!

File: resources/assets/js/components/records/medicals/medical-create-update.vue (hunk 1 above)
Comment (nkeena) [reply resources/assets/js/components/records/medicals/medical-create-update.vue]: Fixed! It's now sending the ID. https://github.com/missions-me/missions/commit/3d72aec2520573b52dc2ece1913380f2ff9f3520

**Comment by jgiovanni** [comment]
It is ready for review

**Pull request merged**